crates/edr_napi/src/logger.rs

```diff
@@ -101,8 +101,22 @@ impl LoggerConfig {
         print_line_callback.unref(env)?;
 
         let print_line_fn = Arc::new(move |message, replace| {
-            let status =
-                print_line_callback.call((message, replace), ThreadsafeFunctionCallMode::Blocking);
+            let (sender, receiver) = channel();
```

Comment:
Should we consider using a `tokio::oneshot::channel` here? Not sure how significant a difference it may be.

Reply:
Tokio's version would require `async` and this is executed in a sync environment.